packages/calcite-components/support/generateT9nDocsJSON.ts

```diff
@@ -24,7 +24,7 @@
         Object.keys(messagesFileMain).forEach((key) => (data[component][key] = {}));
 
         const messagesFilenames = (await readdir(t9nPath, { withFileTypes: true })).map((dirent) => dirent.name);
-        const messagesFilenameRegex = new RegExp(`${component}\\.t9n\\.(.*)\\.json`);
+        const messagesFilenameRegex = new RegExp(`messages\\.(.*)\\.json`);
```

Comment:
Sidenote: should have been caught here https://github.com/Esri/calcite-design-system/pull/11054. 

Is there a way to add a test to assert this?

Reply:
Currently, we don’t have tests for our support files/scripts, but we could explore adding something like output snapshot diffing, similar to the design tokens. Open to ideas here. Could you open a follow-up issue for this?